inst/create-list-available-forecasts.R

```diff
@@ -30,6 +30,7 @@ metrics_quantile <- list(
   "bias" = bias_quantile,
   "coverage_50" = \(...) {run_safely(..., range = 50, fun = interval_coverage_quantile)},
   "coverage_90" = \(...) {run_safely(..., range = 90, fun = interval_coverage_quantile)},
-  "coverage_deviation" = interval_coverage_deviation_quantile
+  "coverage_deviation" = interval_coverage_deviation_quantile,
```

Comment:
why are we storing these as data when they are really a list of functions? Why isn't it just a function?

Reply:
Not sure I completely understand what you're suggesting. Made an issue: #449

Reply:
instead of storing as a data object have this be the return value of some function.